src/quipucordsctl/commands/install.py

```diff
@@ -64,6 +72,19 @@ def get_description() -> str:
     }
 
 
+def setup_parser(parser: argparse.ArgumentParser) -> None:
+    """Add arguments to this command's argparse subparser."""
+    parser.add_argument(
+        "-L",
+        "--no-linger",
```

Comment:
I don't remember if argparse allows to do it easily, but can we have `args.enable_linger` and `--no-linger` switching it to false? Once you hit `if not no_linger` things turn into logic puzzles. AFAIR we already had one mistake in UI caused by this kind of negations in variable names.